intera_interface/src/intera_interface/limb.py

```diff
@@ -138,6 +160,12 @@ def __init__(self, limb="right", synchronous_pub=False):
             queue_size=1,
             tcp_nodelay=True)
 
+        ns_pkn = "ExternalTools/" + limb + "/PositionKinematicsNode/"
+        self._iksvc = rospy.ServiceProxy(ns_pkn + 'IKService', SolvePositionIK)
+        self._fksvc = rospy.ServiceProxy(ns_pkn + 'FKService', SolvePositionFK)
+        rospy.wait_for_service(ns_pkn + 'IKService', 5.0)
+        rospy.wait_for_service(ns_pkn + 'FKService', 5.0)
```

Comment:
That's a really question... @IanTheEngineer?

The IKService on the robot uses SNS IK while IKService in Gazebo uses KDL. We're not sure if FKService exists in the Gazebo simulator code.